.travis.yml

```diff
@@ -9,9 +13,18 @@ install:
   - go get github.com/golang/lint/golint
   - make deps
 
+go:
+  - 1.4
```

Comment:
How about we bump it to 1.5 while we're here? (or do we need to specify? will it use the latest by default?)